README.md

```diff
@@ -101,5 +101,13 @@ You are free to implement the genome classes however you want, and using whateer
 When you have implemented the two (or more) classes, describe the complexity of each operation as a function of the genome size (at the time of the operation), and the size of the TE involved (and when copying, the offset you are copying). Put the description here:
 
 **FIXME: OPERATION COMPLEXITY**
+The big difference in the complexety of arrays and linked lists is the reversed relationship they have in indexing and insertion/deletion operations.
+
+indexing into an array is possible in constant time, however inserting into it requiers recomputing the indicies with is done in linear time.
+
+The opposite is true for linked lists where indexing into it is done in linear time but inserting and deleting from it is done in constant time.
+
+of course this is in regards to single inserts and indexes. in the current implementation we insert TE's nucleotide by nucleotide. the linked lists could very well be improved by inserting the entire TE at once. ie. merging two linked lists.
```

Comment:
This is a little simplistic. It is true that it takes linear time to find an index with a linked list and constant time with an array, and linear time to insert into an array while it takes constant time to insert into a linked list. But what about the operations you have implemented? If I insert a new TE with length $k$ at index $i$, what is the time then? How long does it take to copy a TE that sits at index $i$, is $k$ long, and the copy is moved by an offset $m$?

For insertion, you could say that the array solution takes time $O(1)$ to find the location, time $O(k)$ to create the new TE, and $O(n-i)$ to expand the array (if the genome is $n$ long). A total of $O(k+n-i)$. For the linked list, the time would be $O(i)$ for getting to the index, $O(k)$ for creating the TE, and $O(1)$ for inserting it (if you merge it in; if you insert it one link at a time it takes $O(k)$ but then you don't pay $O(k)$ for creating it). A total of $O(k+i)$.

Try to do the rest yourself, and consider when one solution would be better than another. They both have pros and cons.